app/jobs/fastly_log_processor.rb

```diff
@@ -0,0 +1,75 @@
+require 'zlib'
+FastlyLogProcessor = Struct.new(:bucket, :key) do
+  def perform
+    counts = download_counts
+
+    unless Gemcutter::ENABLE_FASTLY_LOG_PROCESSOR
```

Comment:
This seems like a weird place for the feature flag. Can we move it to where we enqueue the job instead?